mk/tests.mk

```diff
@@ -25,3 +25,7 @@ endef
 print-top-help += \
   echo "  check: Run unit tests"; \
   echo "  installcheck: Run functional tests";
+
+# Variable specifying whether the tests that assume that we have a
+# local build of Nix should run
+HAVE_LOCAL_NIX_BUILD ?= 1
```

Comment:
I don't want to be too obnoxious about this PR :), but `mk/` is supposed to be basically nix-agnostic infrastructure (e.g. stuff that would be chucked entirely if we switched build systems), whereas this is definitely nix-specific and not supposed to be generic infra.

I think this would go better in `Makefile` or `tests/local.mk`.

Reply:
Oooh, that's actually a very good point. Just moved it to `tests/local.mk`

Reply:
Thank you!